spotlight-client/src/contentApi/sources/us_pa.ts

```diff
@@ -181,7 +181,8 @@ const content: TenantContent = {
       name: "Current Prison Population",
       methodology: `<p>
         This data includes all individuals that are currently incarcerated in a DOC
-        facility. It does not include individuals incarcerated in county jails.
+        facility. It does not include individuals incarcerated in county jails. Other
+        includes individuals such as those that are currently out of state.
```

Comment:
Holding on merging this until we see if @ageiduschek identifies other classes to include.